Source/WebKit/UIProcess/Extensions/WebExtension.h

```diff
@@ -147,7 +147,7 @@ class WebExtension : public API::ObjectImpl<API::Object::Type::WebExtension>, pu
     Ref<API::Data> serializeManifest();
 
     double manifestVersion();
-    bool supportsManifestVersion(double version) { return manifestVersion() >= version; }
+    bool supportsManifestVersion(double version) { ASSERT(version > 2); return manifestVersion() >= version; }
```

Comment:
Is there no reason to call this with `supportsManifestVersion(2)`?

Reply:
I mention this in the change log, but passing 2 will be true for every extension.